core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -20,10 +29,10 @@ sealed trait Validator[T] {
   def show: Option[String] = Validator.show(this)
 }
 
-object Validator extends ValidatorMagnoliaDerivation with ValidatorEnumMacro {
+object Validator extends ValidatorEnumMacro with ValidatorMagnoliaDerivation {
   type EncodeToRaw[T] = T => Option[scala.Any]
 
-  private val _pass: Validator[Nothing] = all()
+  private def _pass: Validator[Nothing] = all()
```

Comment:
nothing big, but why does this need to be a `def`?

Reply:
While trying to deal with ambiguous implicits I firstly tried to create a `ValidatorInstances` trait inherited by `Validator` and `ValidatorDerivation`, leading to `NullPointerException`s.

Fortunately `Refute` fixes my problem without these issues, so we can use a `val`.